src/core/Directus/Database/TableGateway/BaseTableGateway.php

```diff
@@ -400,13 +400,13 @@ public function addRecordByArray(array $recordData)
         // photos, which are the only usecase for the thumbnailer.
         // TODO: Move this somewhere more sensible, and don't read the whole file contents into memory
         // in the first place.
-        if (
-            array_key_exists('filename', $recordData) &&
-            array_key_exists('data', $recordData) &&
-            strlen($recordData['data']) > 25000000
-        ) {
-            $recordData = ['filename' => $recordData['filename']];
-        }
+        // if (
```

Comment:
We can get rid of this completely. No need to leave commented out code in the codebase 🙂

Reply:
@rijkvanzanten I removed the commented code. 🙂